app/controllers/users.rb

```diff
@@ -0,0 +1,24 @@
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/user/new' do
+    erb :'user/new'
+  end
+
+  post '/user/create' do
+    user = User.create(name: params[:name], email: params[:email],
+                username: params[:username], password: params[:password],
+                password_confirmation: params[:password_confirmation])
+    session[:username] = user.username
+    if user
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)